src/main/java/de/tum/in/www1/artemis/service/CourseService.java

```diff
@@ -22,16 +23,21 @@
 
     private final Logger log = LoggerFactory.getLogger(CourseService.class);
 
+    private static final double INITIAL_TOTAL_SCORE = 0.0;
+    private static final double SCORE_NORMALIZATION_VALUE = 0.01;
+
     private final CourseRepository courseRepository;
+    private final ParticipationRepository participationRepository;
     private final UserService userService;
     private final ExerciseService exerciseService;
     private final AuthorizationCheckService authCheckService;
 
     public CourseService(CourseRepository courseRepository,
                          UserService userService,
                          ExerciseService exerciseService,
-                         AuthorizationCheckService authCheckService) {
+                         AuthorizationCheckService authCheckService, ParticipationRepository participationRepository) {
         this.courseRepository = courseRepository;
+        this.participationRepository = participationRepository;
         this.userService = userService;
         this.exerciseService = exerciseService;
         this.authCheckService = authCheckService;
```

Comment:
Can you revert these changes as well, if they are not needed?

Reply:
Done :)